src/components/CurrencySymbolButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
This component and `AmountTextInput` are useless IMO. We are never going to reuse this. This can directly be merged into `TextInputWithCurrencySymbol`

Reply:
I also think the same

Reply:
We already had that in the main component but based on https://github.com/Expensify/App/pull/8299#discussion_r855574190 I refactored that into a separate component.

Reply:
Ok, Let's wait for Toglen to pitch in.

Reply:
cc: @tgolen

Reply:
cc: @tgolen

Reply:
@parasharrajat @mdneyazahmad While I agree it is not needed to be split this way to its own components, I think this refactoring helps engineer who is new to this part of codebase to understand quickly what is going on and helps to keep the other components more compact. So I would keep it as it is now.

This is NAB indeed, we can always refactor if we decide to do so.